src/main/webapp/create-recipe-blank.html

```diff
@@ -0,0 +1,211 @@
+										<path fill-rule="evenodd" d="M8 3.5a.5.5 0 0 1 .5.5v4a.5.5 0 0 1-.5.5H4a.5.5 0 0 1 0-1h3.5V4a.5.5 0 0 1 .5-.5z"/>
+										<path fill-rule="evenodd" d="M7.5 8a.5.5 0 0 1 .5-.5h4a.5.5 0 0 1 0 1H8.5V12a.5.5 0 0 1-1 0V8z"/>
+										<path fill-rule="evenodd" d="M8 15A7 7 0 1 0 8 1a7 7 0 0 0 0 14zm0 1A8 8 0 1 0 8 0a8 8 0 0 0 0 16z"/>
+									</svg>
+								</div>
+							</div>
+							<div class="med-sep">
+								<h2>Equipment</h2>
+								<div class="form-group small-sep">
+									<input type="text" class="form-control" id="ingredientInput" placeholder="Start typing for suggestions...">
+								</div>
+								<div class="small-sep">
+									<svg class="bi bi-plus-circle bootstrap-icon" viewBox="0 0 16 16" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
+										<path fill-rule="evenodd" d="M8 3.5a.5.5 0 0 1 .5.5v4a.5.5 0 0 1-.5.5H4a.5.5 0 0 1 0-1h3.5V4a.5.5 0 0 1 .5-.5z"/>
+										<path fill-rule="evenodd" d="M7.5 8a.5.5 0 0 1 .5-.5h4a.5.5 0 0 1 0 1H8.5V12a.5.5 0 0 1-1 0V8z"/>
+										<path fill-rule="evenodd" d="M8 15A7 7 0 1 0 8 1a7 7 0 0 0 0 14zm0 1A8 8 0 1 0 8 0a8 8 0 0 0 0 16z"/>
+									</svg>
+								</div>
+							</div>
+							<div class="med-sep">
+								<h2>Steps</h2>
+								<div class="form-group small-sep">
+									<input type="text" class="form-control" id="ingredientInput" placeholder="Enter a step...">
+								</div>
+								<div class="small-sep">
+									<svg class="bi bi-plus-circle bootstrap-icon" viewBox="0 0 16 16" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
+										<path fill-rule="evenodd" d="M8 3.5a.5.5 0 0 1 .5.5v4a.5.5 0 0 1-.5.5H4a.5.5 0 0 1 0-1h3.5V4a.5.5 0 0 1 .5-.5z"/>
+										<path fill-rule="evenodd" d="M7.5 8a.5.5 0 0 1 .5-.5h4a.5.5 0 0 1 0 1H8.5V12a.5.5 0 0 1-1 0V8z"/>
+										<path fill-rule="evenodd" d="M8 15A7 7 0 1 0 8 1a7 7 0 0 0 0 14zm0 1A8 8 0 1 0 8 0a8 8 0 0 0 0 16z"/>
+									</svg>
+								</div>
+							</div>
+							<div class="med-sep">
+								<h2>Description</h2>
+								<div class="form-group">
+									<label for="descriptionTextArea">Give your recipe some more context.</label>
+									<textarea class="form-control" id="descriptionTextArea" placeholder="Tell us about your recipe..."></textarea>
+								</div>
+							</div>
+							<div class="med-sep">
+								<h2>Video</h2>
+									<label for="videoUploadButton">Add a relevant video accompaniment.</label>
+									<p></p>
+									<button type="button" class="btn btn-primary" id=videoUploadButton>YouTube</button>
+							</div>
+							<div class="med-sep">
+								<h2>Permissions</h2>
+								<div class="form-group">
+									<label for="permissionSelect">Who can see your post?</label>
+									<select class="form-control" id="permissionSelect">
+										<option>Only Me</option>
+										<option>My Groups</option>
+										<option>Anyone</option>
+									</select>
+								</div>
+							</div>
+							<div class="med-sep">
```

Comment:
What does "sep" stand for?

Reply:
Separator (i.e. spacing). I think that looking at the .med-sep in the css file makes this clear. There's also a small-sep, which is half the height of med-sep; it's consistent across the css file.